app/controllers/trainee/profiles_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Trainee::ProfilesController < ApplicationController
+  before_action :trainee_log_in?
+
+  def index
+  end
+
+  def show
+    @current_course_trainee = CourseTrainee.find_by trainee_id: current_trainee.id
```

Comment:
find by trainee_id and status of course

Reply:
Anh ơi trường status ở models thì lại k phải trong bảng CourseTrainee mà ở trong bảng course trong khi không lấy trực tiếp từ bảng Course được nên anh xem giúp em đoạn code em viết trong controller đã được chưa ạ